Example/Podfile.lock

```diff
@@ -72,7 +72,7 @@ DEPENDENCIES:
   - AppHub
   - ARGenericTableViewController
   - Artsy+Authentication/email (from `https://github.com/artsy/Artsy-Authentication.git`, branch `fetch-user-details`)
-  - Artsy+UIFonts
+  - Artsy+UIFonts (from `https://github.com/artsy/Artsy-OSSUIFonts.git`, branch `always_artsy`)
```

Comment:
Seems you’ll have to run `pod install` once more after you removed this from `Podfile`. (PS Also update to latest CocoaPods, in case you are still using `1.2.0-rc`